graphene_linked_events/schema.py

```diff
@@ -810,10 +819,10 @@ def mutate(root, info, **kwargs):
     def _create_p_event(event_obj, organisation, p_event_data) -> PalvelutarjotinEvent:
         p_event_data["linked_event_id"] = event_obj.id
         p_event_data["organisation_id"] = organisation.id
-        p_event_data["organisation_id"] = organisation.id
-        translations = p_event_data.pop("translations")
+        translations = p_event_data.pop("translations", None)
```

Comment:
The event will be created in LinkedEvents before the PalvelutarjotinEvent is created. When the PalvelutarjotinEvent creation was a failure because of missing translations, the event was created only in the LE side.

I added a fix and a test for missing translations, but in new version of the UI, the translations should always be populated, so this should have never risen up. The logs shows that Hurjaruuth user ha an old version of the event creation wizard / form.

Validation before the event would be too much work and just make the mess of code. Lets try to manage with the little fix done.